### PR TITLE
Add E2E Tests for Test Notification Configuration

### PR DESCRIPTION
**Pull request by SOHAMPAL23**
# Pull Requests Requirements

- This PR adds comprehensive E2E tests for the Test Notification Configuration module, ensuring core functionalities are properly validated.

- Notification menu navigation.
- Editing subject and message templates.
- Enabling and disabling notifications.
- Error handling and edge cases.
- Utilized Page Object Model (POM) for better code organization.
- Integrated fixtures to separate test data.

## Summary

- Improves overall test coverage, ensuring that all features in the notification menu are validated, contributing to project stability.
- Having automated tests ensures that new code changes don’t unintentionally break existing functionality, making deployments safer and faster.

## Related Issue

https://github.com/DIGI-UW/OpenELIS-Global-2/issues/1687

### CODE REVIEW

**Review by Bahati308 — COMMENTED** [review]
these tests are not added to the list, cypress wont capture them. Do add them then spin over again

**Comment by SOHAMPAL23** [comment]
Thanks for making me know what I was doing wrong @Bahati308 
Can you help how to add these files so that cypress catches it .Can you tell me how can I go further in this

**Comment by Bahati308** [comment]
head to cypress.config.js file

**Comment by SOHAMPAL23** [comment]
As I was working on this issue https://github.com/DIGI-UW/OpenELIS-Global-2/issues/1619 I changed it into glob spec pattern for  automatically adding the cypress file so will that work like this 

OR 

I should pull the old code and add in that

**Comment by mozzy11** [comment]
@SOHAMPAL23 i see there some CI build failures

**Review by Bahati308 — COMMENTED** [review]
seems the cypress.config.js file is recuptured,

File: frontend/cypress/e2e/AdminE2E/testnotification.cy.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
+import NotificationConfigPage from "../../pages/TestNotification";
+
+describe("Test Notification Config E2E with Page Object Model", () => {
+  before(() => {
+    cy.log("Starting Notification Config E2E Tests 🚀");
+  });
+
+  beforeEach(() => {
+    cy.visit("/MasterListsPage#testNotificationConfigMenu");
```
Comment (Bahati308) [inline frontend/cypress/e2e/AdminE2E/testnotification.cy.js]: this test can not run without logging in first, you could share a recording if it is not the case

Comment (SOHAMPAL23) [reply frontend/cypress/e2e/AdminE2E/testnotification.cy.js]: @Bahati308 Ok I will resolve this problem by  putting the login part
